subprojects/version-control/version-control.gradle.kts

```diff
@@ -39,6 +39,7 @@ dependencies {
     implementation(library("jsch"))
 
     testImplementation(project(":native"))
+    testImplementation(project(":snapshots"))
```

Comment:
Why is this needed?

Reply:
We are using `TestFiles` in those projects, and that type now depends on `FileSystemSnapshotter`. Moreover, over test fixtures don't do transitive dependencies :(

So adding the dependency seems to be the current workaround (see [this Slack discussion](https://gradle.slack.com/archives/C03A0MT6S/p1560348413072200?thread_ts=1560339024.071900&cid=C03A0MT6S)).